src/main/java/io/github/wistefan/mapping/annotations/SubscriptionId.java

```diff
@@ -0,0 +1,14 @@
+package io.github.wistefan.mapping.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to indicate that a certain method should be used for returning the subscription-id
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface SubscriptionId {
```

Comment:
Why does it need a dedicated SubscriptionId annotation? Can an object have an entity and an subscription id?

Reply:
Same here as above